.gitignore

```diff
@@ -14,3 +14,4 @@ __pycache__/
 # Coverage information
 /.coverage
 /htmlcov/
+.pytest_cache/
```

Comment:
This `.gitignore` file only ignores files that might be generated by Pulp Smash or required tooling, such as Python, Sphinx, or coverage. Adhering to this distinction helps avoid the "but what about X cool tool?" discussions. I can add comments to this file to make this clear, if it helps.

Files generated by tools that the user has elected to use can be ignored by the user like so:

```sh
mkdir -p ~/.config/git
cat ~/.config/git/excludesfile <<EOF
# Created by pytest
.pytest_cache/
EOF
touch ~/.config/git/config
git config --global core.excludesfile ~/.config/git/excludesfile
```

Otherwise, this PR looks great.